lib/multi.go

```diff
@@ -123,6 +123,7 @@ func (e *ThreadSafeNucleiEngine) ExecuteNucleiWithOptsCtx(ctx context.Context, t
 			return err
 		}
 	}
+	defer tmpEngine.closeInternal()
 	// create ephemeral nuclei objects/instances/types using base nuclei engine
```

Comment:
@Mzack9999 , the tmpEngine uses shared resources like interactsh,outputwriter etc which are used by every nuclei instance created from threadsafeNucleiExecuter, while fixing memory leaks we accidently closed this here , but this is not expected to be closed here, 

shared resources of threadsafeNucleiExecuter are released using .Close() of `ThreadSafeNucleiEngine` as for temporary resources like ratelimit and stuff they are closed using `defer closeEphemeralObjects(unsafeOpts)`

i have removed this in my last pr and is included in latest, so we should remove this tmpEngine.closeInternal

Reply:
removed